### PR TITLE
Adds README documentation to the project

### PR DESCRIPTION
**Pull request by bkmeneguello**

### CODE REVIEW

**Review by RafaeLeal — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,42 @@
...
+```
+
+## Configuration
+
+| Key | Default | Description |
+| - | - | - |
+| `:api.server/url` | `"http://localhost:8080"` | The Kubernetes REST API endpoint |
+| `:github.oauth-token/path` | `"/etc/github-statuses-updater/oauth-token"` | Path to GitHub OAuth Token |
+| `:tekton/namespace` | `"default"` | Namespace where Tekton Pipelines are deployed |
+| `:tekton-watcher.server/port` | `9000` | Port that Tekton Watcher expose heatlcheck in `/health/live` path |
```
Comment (dankraemer) [inline README.md]:
```suggestion
| `:tekton-watcher.server/port` | `9000` | Port that Tekton Watcher expose healthcheck in `/health/live` path |
```

File: build/containerize.sh [hunk 2]
```diff
@@ -15,7 +15,7 @@ rm -rf $target; mkdir -p $target
 
 ## Generate the manifest of the image to be built
 vessel image --registry docker.io \
-       --repository alangh/tekton-watcher \
+       --repository tekton-watcher \
```
Comment (dankraemer) [inline build/containerize.sh:18]: Will vessel find the correct repository without `alangh/` in the name ?

Comment (bkmeneguello) [reply build/containerize.sh:18]: Vessel will do. There could be issues to deploy the manifest to Kubernetes wich will require modifications outside the scope of documentation.

File: README.md [hunk 3]
```diff
@@ -1 +1,42 @@
...
+
+## Installation
+
+> To build Tekton Watcher it depends on [Vessel](https://github.com/nubank/vessel)
+
+Update `app/*.yaml` configurations to your needs
+
+```
+make build
+docker image load -i target/tekton-watcher.tar
```
Comment (alan-ghelardi) [inline README.md:15]: Just to mention, `Vessel` has a push command that pushes a tarball to a remote registry without loading it into the Docker daemon.

File: README.md [hunk 4]
```diff
@@ -1 +1,42 @@
-# tekton-watcher
+# Tekton Watcher
+
+Tekton Watcher is a service to react to events that occur in Tekton Pipelines and Tasks.
```
Comment (alan-ghelardi) [inline README.md:3]: I wonder if we should to use the terms pipeline runs and task runs since for Tekton pipelines and tasks are merely blueprints.

**Review by alan-ghelardi — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (bkmeneguello) [reply README.md:3]: I thought about that too, my concern is to confuse the user with the specific verbiage

**Review by dankraemer — APPROVED** [review]

**Review by kmoraes — APPROVED** [review]

**Pull request closed without merging**